src/Weave.jl

```diff
@@ -331,8 +341,12 @@ include("display_methods.jl")
 include("reader/reader.jl")
 include("run.jl")
 include("cache.jl")
-include("formats.jl")
-include("format.jl")
+include("rendering/format.jl")
+include("rendering/common.jl")
+include("rendering/htmlformats.jl")
+include("rendering/markdownformats.jl")
+include("rendering/texformats.jl")
+include("rendering/variousformats.jl")
```

Comment:
I prefer to have

> Weave.jl
```julia
include("rendering/rendering.jl")
```

and 
> rendering/rendering.jl
```julia
include("common.jl")
include("htmlformats.jl")
...
```
as I did for `include("reader/reader.jl")`. How about that ?

Reply:
yeah, better keep it consistent